_modules/dotnet-build.test.js

```diff
@@ -0,0 +1,66 @@
+/**************************************************************************************************
+ * Imports
+**************************************************************************************************/
+
+const child_process = require("child_process");
+const dotnetBuild   = require("./dotnet-build");
+const dotnetClean   = require("./dotnet-clean");
+const dotnetPath    = require("./dotnet-path");
+const dotnetRestore = require("./dotnet-restore");
+const faker         = require("faker");
+const shell         = require("shelljs");
+
+describe("dotnetBuild", () => {
+    let dotnetCleanSpy;
+    let dotnetPathSpy;
+    let dotnetRestoreSpy;
+    let shellExitSpy;
+
+    beforeEach(() => {
+        dotnetCleanSpy   = jest.spyOn(dotnetClean, "run").mockImplementation();
+        dotnetPathSpy    = jest.spyOn(dotnetPath, "solutionPathOrExit").mockImplementation();
+        dotnetRestoreSpy = jest.spyOn(dotnetRestore, "run").mockImplementation();
+        shellExitSpy     = jest.spyOn(shell, "exit").mockImplementation();
+    });
+
+    describe("run", () => {
```

Comment:
While the only method tested currently, recommend surrounding with our region/endregion comments so the next traveler through is inclined to follow that